microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -159,19 +163,31 @@ For legacy reference, the following format was used for User-Agent string.
 On Windows, the `User-Agent` HTTP request header uses the following format:
 
 ```https
-User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Safari/537.36 Edg/90.0.818.46
+User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64)  
+    AppleWebKit/537.36 (KHTML, like Gecko)  
+    Chrome/90.0.4430.85  
+    Safari/537.36  
+    Edg/90.0.818.46

```

Comment:
I don't think you should indent these subsequent lines with 4 spaces.

Reply:
i was trying to get the following lines to remain in black text ... is there a way to do this with a newline?

Reply:
I think this happens because the code block uses the `https` language setting. I think we should drop this language entirely (just use 3 backtick characters, and remove the `https` word behind them).
In fact, I would also remove the `User-Agent: ` prefix in this code block. This is not what we care about here. It's the name of the http header, but what we want to document here is the value.

Reply:
good idea, fixed